composer.json

```diff
@@ -3,7 +3,8 @@
     "description": "Gebrüder Weiss Woocommerce Plugin",
     "type": "wordpress-plugin",
     "require": {
-        "guzzlehttp/guzzle": "^7.3"
+        "guzzlehttp/guzzle": "*",
```

Comment:
I would not use any as a version constraint, since this could lead to unexpected major version updates.